lib/doorkeeper/rails/helpers.rb

```diff
@@ -11,11 +11,11 @@ def doorkeeper_authorize!(*scopes)
         end
       end
 
-      def doorkeeper_unauthorized_render_options
+      def doorkeeper_unauthorized_render_options(error)
```

Comment:
Unused method argument - `error`. If it's necessary, use `_` or `_error` as an argument name to indicate that it won't be used. You can also write as `doorkeeper_unauthorized_render_options(*)` if you want the method to accept any arguments but don't care about them.